tests/staggeredrenderer/test_staggeredrenderer.cpp

```diff
@@ -53,10 +53,10 @@ void test_StaggeredRenderer::mapSize()
     StaggeredRenderer renderer(mMap);
 
     TileLayer *tileLayer = static_cast<TileLayer*>(mMap->layerAt(0));
-    QSize mapSize = renderer.mapSize();
+    QSize mapSize = renderer.mapBoundingRect().size();
```

Comment:
To make the test a little more complete, just take the bounding rect and compare it to a `QRect(0, 0, 10 * 64 + 32, 10 * 16 + 16)`.